File: sprint4/By languages/C#/lesson1code1.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        Dictionary<string, string> countryByCity = new Dictionary<string, string>()
        {
            {"Moscow", "Russia"},
            {"Tula", "Russia"},
            {"London", "UK"},
            {"Paris", "France"},
            {"Leon", "France"},
            {"San Francisco", "USA"},
            {"New York", "USA"}
        };

        // Присвоить значение по ключу
        countryByCity["Saint Petersburg"] = "Russia";

        // Получить значение по ключу
        string country;
        countryByCity.TryGetValue("Tula", out country);
        Console.WriteLine("Tula is located in " + country);
    }
}

File: sprint4/By languages/go/lesson1code1.go
package main

import "fmt"

func main() {
    countryByCity := map[string]string{
        "Moscow":        "Russia",
        "Tula":          "Russia",
        "London":        "UK",
        "Paris":         "France",
        "Leon":          "France",
        "San Francisco": "USA",
        "New York":      "USA",
    }

    // Присвоить значение по ключу
    countryByCity["Saint Petersburg"] = "Russia"

    // Получить значение по ключу
    country := countryByCity["Tula"]
    fmt.Printf("Tula is located in %s\n", country)
}
